doc/manual.md

```diff
@@ -44,6 +44,20 @@ Before we get too far, let's define some of the terminology used in this documen
 The reasoning behind our definition of "sample" may not be obvious to all readers. If the time to execute a benchmark is smaller than the resolution of your timing method, than a single evaluation of the benchmark will generally not produce a valid sample. In that case, one must approximate a valid sample by
 recording the total time `t` it takes to record `n` evaluations, and estimating the sample's time per evaluation as `t/n`. For example, if a sample takes 1 second for 1 million evaluations, the approximate time per evaluation for that sample is 1 microsecond. It's not obvious what the right number of evaluations per sample should be for any given benchmark, so BenchmarkTools provides a mechanism (the `tune!` method) to automatically figure it out for you.
 
+## The difference between real-time and cpu-time
+BenchmarkTools measures both real-time and cpu-time. Real-time is the time that has passed while running your program and cpu-time is the time
+that a processor has spend executing your program. As an example when the operating system deschedules your program, real-time is still passing,
+but cpu-time is paused, this is also the case when your program spends time in the kernel or is blocked waiting. As an example take a program that
```

Comment:
This is not quite correct. Since cpu-time will increase with kernel work, but will not increase if the kernel puts the process to sleep.